contracts/bancor-portal/BancorPortal.sol

```diff
@@ -271,28 +277,32 @@ contract BancorPortal is IBancorPortal, ReentrancyGuardUpgradeable, Utils, Upgra
         // look for relevant whitelisted pools, revert if there are none
         bool[2] memory whitelist;
```

Comment:
Do the conversion weth-to-native once (on declaration line), then you don't need to keep calling this function:
```
Token[2] memory tokens = [_wethToNative(token0), _wethToNative(token1)];
```

Reply:
done